src/pages/GameNew.js

```diff
@@ -91,9 +94,20 @@ const GameNew = ({ createGame }) => {
               style={{ height: "5rem", width: "18rem", fontSize: "18px" }}
             />
           </FormGroup>
+          <FormGroup>
+            <Label for="user_id" hidden>
+              User Id
+            </Label>
+            <Input
+              id="user_id"
+              name="user_id"
+              onChange={handleChange}
+              value={newGame.user_id || currentUser?.id}
```

Comment:
Question: is this the more appropriate syntax to use? I am learning so I would appreciate knowing.

Reply:
we are using the || as a fallback so if newGame.user_id is falsy (e.g., null, undefined, or an empty string), it will use currentUser?.id. So far we have found this method to work . We were having issues creating a game in our app unless the user_id was added so we took extra measures to to ensure we were able to grab it on creation. Since we got the functionality we intended we didn't go back for a refactor of the code yet.

Reply:
Updated the files to remove the console.logs. Still have one left for the delete functionality because delete function works but throws an error.  The error doesn't affect delete functionality because it works and item is deleted after an index page refresh. So we are working on the error.

Reply:
Updated and re-added console.logs for fetch functionality.